README.md

```diff
@@ -107,19 +107,20 @@ the EDR service.
 The spatial extent needs to be created based upon the type of query
 type. To set the spatial extent, the ‘Set spatial extent’ button should
 be clicked. This will open a dialogue box that will vary according to
-the data query type. A polygon will be created for an area query, a
-point for a position query and a circle radius based on a centre point
-for the radius query for example. These are shown in detail below.
+the data query type. A polygon will be created for an area and cube query, a
+point for a position query, a line string for the corridor and trajectory queries and a circle radius based on a centre point
+for the radius query. These are defined in detail below.
 
 Should the spatial extent not be set, and the user attempts to run the
 query, then there will be a warning message before the user is prompted
 to select the extent after clicking OK.
 
 <img src="./docs/images/image6.png"
-style="width:2.06358in;height:0.70871in" alt="A screenshot of a computer
-Description automatically generated" />
+style="width:2.06358in;height:0.70871in" alt="Warning of the spatial extent not being set" />
+
+The user will then be given an option to input the extent for the type of query selected based upon the input dialogues listed below.
 
-The user will then be given an option to input the extent for the query.
+**NOTE: the corridor and the trajectory queries are experimental services and may experience issues based upon the EDR server configuration. They may not respond well to large quantities of data being returned by an EDR service.**
```

Comment:
All EDR queries, perhaps with the exception of trajectory and point, which are expected to return a large amount of data could suffer from slow responses.  For potentially large data requests (large geospatial area, number of levels, timesteps, parameters, high data resolution), users should choose a binary format such as netCDF or GRIB2 (if available). JSON, CoverageJSON and GeoJSON are verbose and can add delay to requests. This can attributed to the time to respond to the request from the server, the time to transmit the data, or the time for QGIS to process and render the data payload.

Reply:
This has been updated now and is on the latest PR.